kernel/src/kcb.rs

```diff
@@ -355,6 +364,10 @@ impl<A: ArchSpecificKcb> Kcb<A> {
         self.physical_memory.gmanager = Some(gm);
     }
 
+    pub fn set_global_pmem(&mut self, gm: &'static GlobalMemory) {
+        self.pmem_memory.gmanager = Some(gm);
+    }
+
     pub fn set_allocation_affinity(&mut self, node: atopology::NodeId) -> Result<(), KError> {
```

Comment:
set_mem_affinity for namign consistency?

Reply:
Updated.